website/src/content/development/layout-guidelines.md

```diff
@@ -0,0 +1,122 @@
+# Layout Guidelines
+
+Layouts are the foundation for interactive experiences. They affect the way data is structured and displayed. The different layout types set the minimum and maximum width of a container and its position on the page.
+
+### Design
+
+In commercetools, we always try to make better use of the space when building Merchant Center applications. As a result, we provide guidelines and rules that are recommended to be followed when developing applications, both by internal and external engineering teams. You can choose from 5 different layout options to display content in the Merchant Center:
```

Comment:
Since this is a public docs, I would probably not mention internal teams.
```
As a result, we provide guidelines and rules that are recommended to be followed, when developing applications, by engineering teams.
```